official/vision/beta/projects/yolo/modeling/tests/test_CSPDownSample.py

```diff
@@ -0,0 +1,52 @@
+import tensorflow as tf
```

Comment:
I think we should put tests together with the library. Please check out the convention of the existing code base.
E.g. modeling/csp_down_sample_test.py

Reply:
Fixed in a0dd0e8